### ISSUE TITLE
Add all the sounds

### ISSUE THREAD

**Issue by marcus8448** [StellarHorizons/Galacticraft-Rewoven#8]
Entity, boss, airlock, player, deconstructor, music, and rocket sounds need to be ported

**Comment by craftablescience** [comment]
I've just done this, I think all the music is in it
Let me check before this is closed

**Comment by craftablescience** [comment]
Yep, all music is included. When I get back to a desktop computer I will change galacticraft to galacticraft-fabric in sounds.json, but I want to verify that's valid first.

**Comment by vos6434** [comment]
We have new sounds for mobs and possibly more.

**Comment by TakeABao23** [comment]
Can now track progress here:
[Galacticraft 5 Sounds](https://github.com/orgs/TeamGalacticraft/projects/10)
